CONTRIBUTING.md is outdated

First of all, there are no clear instructions how to test changes in the derive macro. What steps is necessary to run all testsuites?

- There is no folder `test_suite/deps`, which is [required](https://github.com/serde-rs/serde/blob/master/CONTRIBUTING.md#in-the-test_suitedeps-directory) to run tests

- If I try to run tests on stable rust from the repository root, I get the error:
  ```
  d:\Projects\serde>cargo test
    Compiling serde_test_suite v0.0.0 (D:\Projects\serde\test_suite)
  error[E0658]: non-ascii idents are not fully supported
    --> test_suite\tests\test_gen.rs:271:9
      |
  271 |         σ: f64,
      |         ^
      |
      = note: see issue #55467 <https://github.com/rust-lang/rust/issues/55467> for more information

  error: aborting due to previous error

  For more information about this error, try `rustc --explain E0658`.
  error: could not compile `serde_test_suite`

  To learn more, run the command again with --verbose.

  d:\Projects\serde>
  ```
  This error is quite strange, because [the referenced issue](https://github.com/rust-lang/rust/issues/55467) is closed.

- I was only able to ran tests when I supply the `"unstable"` feature flag, which requires the nightly rust:
  ```
  cargo +nightly test --features unstable
  ```
  Is it possible to run tests on stable?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -34,22 +34,16 @@ tests for you.
 cargo test --features derive
 ```
 
-##### In the [`test_suite/deps`] directory
-
-```sh
-# This is a prerequisite for running the full test suite
-cargo clean && cargo update && cargo build
-```
-
 ##### In the [`test_suite`] directory
 
 ```sh
 # Run the full test suite, including tests of unstable functionality
-cargo test --features unstable
+cargo +nightly test --features unstable
 ```
 
+Note that this test suite currently only supports running on a nightly compiler.
+
 [`serde`]: https://github.com/serde-rs/serde/tree/master/serde
-[`test_suite/deps`]: https://github.com/serde-rs/serde/tree/master/test_suite/deps
 [`test_suite`]: https://github.com/serde-rs/serde/tree/master/test_suite
 
 ## Conduct

